### DISCUSSION TITLE
Insufficient disk space

### DISCUSSION THREAD

**Discussion by mrgraff-dev** [iterate-ch/cyberduck#15423] [Q&A] [answered]
Hello,
 
I am seeing the following error whenever I attempt to edit or upload files to my website:

``` 
Insufficient disk space
552 Disk full - please upload later. Please contact your web hosting service provider for assistance.
``` 
 
To be clear, I know what a 552 error is and I have already contacted my hosting provider. I do not have insufficient space (unlimited storage with my host) the disk is not full and I am able to edit and upload files using other (much slower and clunkier) FTP apps - no errors.

As further proof that I should not be seeing this message, I have Perl scripts that are currently reading, writing, and deleting files.
 
I found a similar discussion from April in which the solution appeared to be delete the Mac log files. However, that did not solve this problem for me. I also sent a similar email to cyberduck support, and they directed me to the forums, clearly not reading the details that I included.

Cyberduck 8.7.1
macOS 12.7.2
 
Thank you,

**Comment by dkocher** [comment, marked as answer]
The _552 Disk full_ reply is sent by the server and thus your server administrator should be able to tell you what the reason is for the disk space error to be returned.

**Reply by mrgraff-dev** [reply]
So, you're trying to tell me that the other FTP clients are ignoring the error, and writing and deleting files on my server anyway? C'mon... no need to insult my intelligence.That said, Cyberduck seems to be magically working for me now;  but I had to spend money on a working solution during the previous 3 weeks to get my work done.

Cyberduck is still a much more pleasant app to work with and I will continue to do so, but I'm not happy about this.

**Reply by dkocher** [reply]
> So, you're trying to tell me that the other FTP clients are ignoring the error…

Not saying that but the sequence of commands from the other clients may be slightly different that would not trigger the server error.
